Cargo.toml

```diff
@@ -11,11 +11,12 @@ tauri = "1.0.0-beta.8"
 thiserror = "1.0"
 serde = "1.0"
 serde_json = "1.0"
-sqlx = { version = "0.5", features = [ "runtime-tokio-rustls", "json" ] }
-tokio = { version = "1.14", features = [ "sync" ] }
+sqlx = { version = "0.5", features = ["runtime-tokio-rustls", "json"] }
+tokio = { version = "1.14", features = ["sync"] }
 futures = "0.3"
 
 [features]
 sqlite = ["sqlx/sqlite"]
 mysql = ["sqlx/mysql"]
 postgres = ["sqlx/postgres"]
+default = ["sqlite"]
```

Comment:
if you don't specify a default, then Rust doesn't provide strong type checking but I believe adding this takes nothing away from the flexibility that a consumer would get from this. Maybe there's another or better way? Seemed to work.

Reply:
seems there are problems with the tests due to this change but hoping someone has a view on how we can optimize this so not only are we testing all DB types but that those cloning the repo will get type support without needing to make changes.

Reply:
ahhh; didn't understand there was a need to negate this with `default-features: false` ... I would agree this is a poor trade off.

Reply:
this has been removed